modules/installing-gcp-user-defined-labels-and-tags.adoc

```diff
@@ -49,25 +47,27 @@ User-defined labels and {product-title} labels are available on the following GC
 
 .User-defined tags
 
-User-defined tags are attached to resources created by the {product-title} Image Registry Operator and not on the resources created by any other Operators or the Kubernetes in-tree components.
+User-defined tags are applied only to resources created by {product-title} installation program and its core components such as:
```

Comment:
Per the ISG, we should try to introduce lists with complete sentences. In this case, I think this might work, but replace "resources" with another word if it is not technically accurate:
```suggestion
User-defined tags are applied only to resources created by {product-title} installation program and its core components, such as the following resources:
```